hub/api/dataset.py

```diff
@@ -38,12 +42,22 @@
     UserNotLoggedInException,
     TokenPermissionError,
 )
+from hub.compression import (
+    IMAGE_COMPRESSIONS,
+    VIDEO_COMPRESSIONS,
+    AUDIO_COMPRESSIONS,
+)
 from hub.util.storage import get_storage_and_cache_chain, storage_provider_from_path
 from hub.util.compute import get_compute_provider
 from hub.util.remove_cache import get_base_storage
 from hub.util.cache_chain import generate_chain
 from hub.core.storage.hub_memory_object import HubMemoryObject
 
+IMAGE_COMPRESSIONS = IMAGE_COMPRESSIONS.copy()
```

Comment:
maybe you can do something like _image_compressions in hub/htype.py instead of this. https://github.com/activeloopai/Hub/blob/main/hub/htype.py#L208

Reply:
Ah! That makes more sense. Forgot you use upper case only for constants.

Reply:
No, I meant you need to include the other compression aliases other than jpg, like in the mentioned file

Reply:
I removed "dcm" like in htype.py as well. Hope this works.